script/vsts/platforms/windows.yml

```diff
@@ -5,10 +5,22 @@ jobs:
     strategy:
       maxParallel: 2
```

Comment:
`maxParallel: 2` might slow things down.

But if we give it too high a limit, we might run into our "10 at once account-wide" limit faster. Maybe bump this to `3`?

Reply:
I will set it to 4 once this gets ready. Azure will automatically queue them for us if we hit the limit. That is nothing to be worried about.